### PR TITLE
Google/quick add

### PR DESCRIPTION
**Pull request by Gautime**
Added the quickadd feature, in which a user can create an event starting from the time slashcommand is called to one hour in duration, using a very simple slashcommand `/calendar quickadd {title of the event}`.

### CODE REVIEW

File: helpers/GCResult.ts [hunk 1]
```diff
@@ -3,20 +3,20 @@ import { HttpStatusCode, IHttp, ILogger, IRead } from '@rocket.chat/apps-engine/
 import { IApiRequest } from '@rocket.chat/apps-engine/definition/api';
 
 export class GCResults {
-    public atoken: string;
+    public acess_token: string;
```
Comment (d-gubert) [inline helpers/GCResult.ts]:
```suggestion
    public access_token: string;
```

Comment (d-gubert) [reply helpers/GCResult.ts]: I mistakenly applied the suggestion; the variable needs to be renamed on the usages as well as in this line.

Reverted for proper alteration

Comment (d-gubert) [reply helpers/GCResult.ts]: @Gautime has this been solved

Comment (Gautime) [reply helpers/GCResult.ts]: Yeah in this new PR which I will soon open for refresh tokens, I have completely replaced this file with another function. So this variable name will not matter.

**Pull request merged**